### PR TITLE
RavenDB-9122 & RavenDB-9123

### PR DESCRIPTION
**Pull request by grisha-kotler**
http://issues.hibernatingrhinos.com/issue/RavenDB-9122
http://issues.hibernatingrhinos.com/issue/RavenDB-9123

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts [hunk 1]
```diff
@@ -413,22 +415,39 @@ class serverDashboard extends viewModelBase {
...
             const start = this.startUpTime();
             const dateFormat = "YYYY MMMM Do, h:mm A";
             return start ? start.local().format(dateFormat) : "";
-        })
+        });
+
+        this.node = ko.pureComputed(() => {
+            const topology = this.clusterManager.topology();
+            const nodeTag = topology.nodeTag();
+            const node = topology.nodes().filter(x => x.tag() === nodeTag)[0];
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts:428]: return topology.nodes().find(x => x.tag() === nodeTag);
and remove if (!node) {
             return null;
            }

            return node;

File: src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts [hunk 2]
```diff
@@ -413,22 +415,39 @@ class serverDashboard extends viewModelBase {
...
     compositionComplete() {
         super.compositionComplete();
-        
+
         this.initSections();
         
         this.enableLiveView();
     }
 
+    private getNode() {
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts:445]: code duplication

File: src/Raven.Studio/wwwroot/App/views/resources/serverDashboard.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div id="dashboard-container" class="content-margin">
+<div id="dashboard-container" class="content-margin cluster">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/resources/serverDashboard.html:1]: it isn't cluster page, Why cluster class here?

File: src/Raven.Studio/wwwroot/App/views/resources/serverDashboard.html [hunk 4]
```diff
@@ -43,19 +43,31 @@ <h3 class="panel-title"><i class="icon-processor"></i> <span>CPU & MEMORY</span>
...
+                                    (<strong data-bind="text: ravenCpuUsage().toFixed(2) + '%', attr: { class: ravenCpuUsageClass() }" data-placement="bottom" data-toggle="tooltip" title="RavenDB CPU usage"></strong>)
+                                </div>
                             </div>
                             <div class="infogrid-graph" >
                                 <div class="graph-container fix-size" id="cpuChart">
                                     
                                 </div>
                             </div>
-                            <div class="infogrid-label" data-bind="with: resources">
+                            <div class="infogrid-label" data-bind="with: resources" style="padding: 0">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/resources/serverDashboard.html:62]: please avoid inline css rules.

File: src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts:428]: done

File: src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts (hunk 2 above)
Comment (grisha-kotler) [reply src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts:445]: removed

File: src/Raven.Studio/wwwroot/App/views/resources/serverDashboard.html (hunk 3 above)
Comment (grisha-kotler) [reply src/Raven.Studio/wwwroot/App/views/resources/serverDashboard.html:1]: removed

File: src/Raven.Studio/wwwroot/App/views/resources/serverDashboard.html (hunk 4 above)
Comment (grisha-kotler) [reply src/Raven.Studio/wwwroot/App/views/resources/serverDashboard.html:62]: done

**Pull request merged**